Corona-Warn-App/src/main/res/navigation/nav_graph_onboarding.xml

```diff
@@ -47,6 +38,15 @@
             android:id="@+id/action_onboardingPrivacyFragment_to_onboardingTracingFragment"
             app:destination="@id/onboardingTracingFragment" />
     </fragment>
+    <fragment
+        android:id="@+id/onboardingTracingFragment"
+        android:name="de.rki.coronawarnapp.ui.onboarding.OnboardingTracingFragment"
+        android:label="OnboardingTracingFragment"
+        tools:layout="@layout/fragment_onboarding_tracing">
+        <action
+            android:id="@+id/action_onboardingTracingFragment_to_onboardingTestFragment"
+            app:destination="@id/onboardingTestFragment" />
+    </fragment>
```

Comment:
how is it related to dialog change ?

Reply:
I just rearanged the fragments order in the file so we have better readability. Each fragment in the flow is now one after the other in here as well.